README_ja.md

```diff
@@ -4,25 +4,237 @@ Unit Manager and Cell ManagerはPersoniumのセル単位での管理を行う事
 - Unit Manager - Personium Unit配下のセル（複数）  
 - Cell Manager - 一般セル
 
-## 必要な情報  
-
-1. Personium Unitもしくは一般セル  
-1. セル名（Unitセルもしくは一般セル）  
-1. 上記のセルの管理者アカウントのログイン情報（ID/パスワード）  
 
 ## セットアップ  
-Personium Project's Unit/Cell Managerを使いたい場合、下記のURLにアクセスしてください。   
 
-    https://demo.personium.io/app-uc-unit-manager/__/html/login.html
+### Unit Manager セットアップ手順
+
+この手順では以下の情報を使用します。環境に合わせて修正してください。
+
+* {UnitFQDN}　　 ：Personium Unit URL
+* {AccessToken}　：トランスセルトークン
+
+
+1. [unit-manager.zip](https://github.com/personium/app-uc-unit-manager/raw/master/unit-manager.zip) をダウンロードします。
+1. [Cell作成API](https://personium.io/docs/ja/apiref/current/100_Create_Cell.html)を実行し、自身のPersonium Unitにapp-uc-unit-managerセルを作成します。
+
+    ```console
+    # curl "https://{UnitFQDN}/__ctl/Cell" -X POST -i \
+    -H 'Authorization: Bearer {AccessToken}' -H 'Accept: application/json' \
+    -d '{"Name":"app-uc-unit-manager"}'
+    ```
+
+1. [Cellスナップショットファイル登録更新API](https://personium.io/docs/ja/apiref/current/503_Register_and_Update_Snapshot_Cell.html)を実行し、作成したセルにzipファイルをアップロードします。
+
+    ```console
+    # curl \
+    "https://{UnitFQDN/app-uc-unit-manager/__snapshot/app-uc-unit-manager.zip" \
+    -X PUT -i -H 'Authorization: Bearer {AccessToken}' \
+    -H 'Accept: application/json' \
+    -T "{zip格納フォルダ}/app-uc-unit-manager_cell.zip"
+    ```
+
+1. [CellインポートAPI](https://personium.io/docs/ja/apiref/current/507_Import_Cell.html)を実行し、セルインポートを実行します。
+
+    ```console
+    # curl "https://{UnitFQDN}/app-uc-unit-manager/__import" -X POST -i \
+    -H 'Authorization: Bearer {AccessToken}' \
+    -d '{"Name":"app-uc-unit-manager.zip"}
```

Comment:
"." can not be used for the "Name" value of the body of the cell import.

Reply:
Fixed "Name" value.